esx_service/utils/vmdk_utils.py

```diff
@@ -346,7 +346,11 @@ def find_dvs_volume(dev):
     # "[<datastore name>] <parent-directory>/tenant/<vmdk-descriptor-name>"
     # Trim the datastore name and keep disk path.
     datastore_name, disk_path = dev.backing.fileName.rsplit("]", 1)
-    logging.info("backing disk name is %s", disk_path)
+    readable_vmdk_path = vmdk_ops.gets_readable_path(disk_path)
+    if readable_vmdk_path == '':
+        logging.warning("Failed to fetch readable vmdk path")
+
+    logging.info("backing disk name is %s", readable_vmdk_path)
```

Comment:
At this point, if readable_vmdk_path is empty, we will print warning on 351 but empty vmdk_path. This isn't worth. Lets print the whatever vmdk_path we have if we failed to convert it to readable one